wrims-ide/src/main/java/gov/ca/water/wrims/gui/ide/debugger/listener/WelcomeViewListener.java

```diff
@@ -119,7 +119,7 @@ public void run(){
     }
     
     private void logInfo(String message) {
-        IStatus status = new Status(IStatus.INFO, "gov/ca/water/wrims/gui/ide", message);
+        IStatus status = new Status(IStatus.INFO, "gov.ca.water.wrims.gui.ide", message);
```

Comment:
Other examples of similar: I think this constant is ok, it would be better if it could be captured in a class though, but I don't see an appropriate class to use.
https://github.com/search?q=repo%3ACentralValleyModeling%2Fwrims-gui+PLUGIN_ID&type=code